examples/run/cuda/seeding_example_cuda.cpp

```diff
@@ -6,79 +6,165 @@
+#include "traccc/finding/finding_algorithm.hpp"
+#include "traccc/fitting/fitting_algorithm.hpp"
 #include "traccc/io/read_geometry.hpp"
+#include "traccc/io/read_measurements.hpp"
 #include "traccc/io/read_spacepoints.hpp"
 #include "traccc/io/utils.hpp"
 #include "traccc/options/common_options.hpp"
+#include "traccc/options/finding_input_options.hpp"
 #include "traccc/options/handle_argument_errors.hpp"
+#include "traccc/options/propagation_options.hpp"
 #include "traccc/options/seeding_input_options.hpp"
 #include "traccc/performance/collection_comparator.hpp"
 #include "traccc/performance/timer.hpp"
+#include "traccc/resolution/fitting_performance_writer.hpp"
 #include "traccc/seeding/seeding_algorithm.hpp"
 #include "traccc/seeding/track_params_estimation.hpp"
 
 // Detray include(s).
 #include "detray/core/detector.hpp"
 #include "detray/detectors/toy_metadata.hpp"
 #include "detray/io/common/detector_reader.hpp"
+#include "detray/propagator/navigator.hpp"
+#include "detray/propagator/propagator.hpp"
+#include "detray/propagator/rk_stepper.hpp"
 
 // VecMem include(s).
 #include <vecmem/memory/cuda/device_memory_resource.hpp>
 #include <vecmem/memory/cuda/host_memory_resource.hpp>
+#include <vecmem/memory/cuda/managed_memory_resource.hpp>
 #include <vecmem/memory/host_memory_resource.hpp>
 #include <vecmem/utils/cuda/async_copy.hpp>
-
-// ACTS include(s).
-#include <Acts/Definitions/Units.hpp>
+#include <vecmem/utils/cuda/copy.hpp>
 
 // System include(s).
 #include <exception>
 #include <iomanip>
 #include <iostream>
 
+using namespace traccc;
 namespace po = boost::program_options;
 
-int seq_run(const traccc::seeding_input_config& i_cfg,
+int seq_run(const traccc::seeding_input_config& /*i_cfg*/,
+            const traccc::finding_input_config& finding_cfg,
+            const traccc::propagation_options<scalar>& propagation_opts,
             const traccc::common_options& common_opts, bool run_cpu) {
 
+    /// Type declarations
+    using host_detector_type = detray::detector<detray::toy_metadata<>>;
+
+    using device_detector_type =
+        detray::detector<detray::toy_metadata<>, covfie::field_view,
+                         detray::device_container_types>;
+
+    using b_field_t = typename host_detector_type::bfield_type;
+    using rk_stepper_type =
+        detray::rk_stepper<b_field_t::view_t, traccc::transform3,
+                           detray::constrained_step<>>;
+    using host_navigator_type = detray::navigator<const host_detector_type>;
+    using host_fitter_type =
+        traccc::kalman_fitter<rk_stepper_type, host_navigator_type>;
+    using device_navigator_type = detray::navigator<const device_detector_type>;
+    using device_fitter_type =
+        traccc::kalman_fitter<rk_stepper_type, device_navigator_type>;
+
     // Memory resources used by the application.
     vecmem::host_memory_resource host_mr;
     vecmem::cuda::host_memory_resource cuda_host_mr;
+    vecmem::cuda::managed_memory_resource mng_mr;
```

Comment:
Just fyi, you don't have to use managed memory with the detector anymore

Reply:
Thant can be followed in the later PRs